container-tests/tests/build.gradle

```diff
@@ -23,7 +23,11 @@ dependencies {
   testImplementation 'org.junit.jupiter:junit-jupiter'
   testImplementation 'org.testcontainers:testcontainers'
   testImplementation 'org.web3j:core'
-  testImplementation 'org.web3j:quorum'
+  testImplementation('org.web3j:quorum') {
+    constraints {
+      implementation('org.web3j:core:4.9.4') { because 'quorum 4.9.4 hasn\'t been released yet' }
```

Comment:
```suggestion
      implementation('org.web3j:core:4.9.4') { 
        because 'Web3J/Quorum only has a 4.9.2 lib, but we want it to use web3j libs version 4.9.4 ')
      }
```